universal-application-tool-0.0.1/app/models/Account.java

```diff
@@ -64,6 +70,14 @@ public void setManagedByGroup(TrustedIntermediaryGroup group) {
     return Optional.ofNullable(this.managedByGroup);
   }
 
+  public ImmutableList<Program> getAdministeredPrograms() {
+    return ImmutableList.copyOf(this.administeredPrograms);
+  }
+
+  public void setAdministeredPrograms(ImmutableList<Program> administeredPrograms) {
```

Comment:
Can you do "addAdministeredProgram" (maybe in addition?)

Reply:
Done!